src/Nova/User.php

```diff
@@ -34,7 +34,7 @@ public function subtitle()
         'id',
         'first_name',
         'last_name',
-        'email',
+        //  'email',    // TODO -- needs to be custom search since emails are no longer in users table
```

Comment:
@drewroberts - this should also be tasked out.  If user search by email in Nova is still desired, its implementation needs to change to search on the email_addresses.email field.

Reply:
We don't need that. It will be easy to search for an email address in that nova resource